gems/aws-sdk-core/spec/sigv4_helper.rb

```diff
@@ -1,29 +0,0 @@
-module Sigv4Helper
```

Comment:
Made this more generic, but most of this code was mocking CRT signing (for sigv4a) and stopped working when I made the refactor.